### ISSUE TITLE
Race/Ethnicity Updates: crosswalk analysis between the revised SPD 15 and our current race and ethnicity pattern guidance

### ISSUE THREAD

**Issue by sarah-sch** [uswds/uswds-site#2655]
This issue captures the work of crosswalk analysis between the revised SPD 15 and USWDS's current race and ethnicity pattern guidance

[Race and ethnicity pattern: Crosswalk analysis doc](https://docs.google.com/document/d/1iAJkHR-rPvx_97zY9S4TtMm8bAQFqAU52I_S9o3sZTA)

**Comment by finekatie** [comment]
Analysis is finished and recommendations/suggestions made: https://docs.google.com/document/d/1iAJkHR-rPvx_97zY9S4TtMm8bAQFqAU52I_S9o3sZTA

This is ready for Fed Review.

**Comment by finekatie** [comment]
Anne finished review on 5/22. Still need Dan to review.

**Comment by annepetersen** [comment]
Both Dan and Anne's review is now complete: passing along to our next reviewer

**Comment by finekatie** [comment]
Anne has handed off to OMB after other stakeholder review.

**Comment by finekatie** [comment]
I think this particular task is complete. I think we need a new ticket for updating the documentation based on stakeholder and OMB feedback. @brunerae  and @annepetersen , what do you think?

**Comment by finekatie** [comment]
7/23 - Anne and I discussed complexities about gender and ethnicity and how to comment, post OMB review. OES was contacted for some guidance on collecting non-statistical information. 
8/20 - Discussed with Anne during fed sync. Needed to do a little more research on PRA/OIRA/SPD around that language.

Today, Anne and I worked out some copy in the "What to do" section. This is now ready for their review and final review with Dan. 
We're using [THIS VERSION](https://docs.google.com/document/d/1LE7l8jxz4pDCAYPKleJSZem9TE7yYkbPTL57e17bZMA/edit) for final review.
